nemo_skills/pipeline/utils/backends.py

```
@@ -0,0 +1,309 @@
+
+            def __enter__(self):
+                return self.exp
+
+            def __exit__(self, exc_type, exc, tb):
+                return False
+
+        return _ExperimentCtx(exp_or_name)
+
+    try:
+        return run.Experiment.from_title(exp_or_name)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)